docs/source/docker/advanced/datapool.rst

```diff
@@ -67,6 +67,27 @@ first time.
     |-- passageway1-c1.avi
     `-- terrace1-c0.avi
 
+Let's create a Lightly dataset which uses that bucket (choose your tab - S3, GCS or Azure):
+
+.. tab-set::
+  .. tab-item:: AWS S3 Datasource
+
+    .. literalinclude:: ./code_examples/python_create_dataset_s3_example.py
+      :linenos:
+      :language: python
+
+  .. tab-item:: GCS Datasource
+
+    .. literalinclude:: ./code_examples/python_create_dataset_gcs_example.py
+      :linenos:
+      :language: python
+
+  .. tab-item:: Azure Datasource
+
+    .. literalinclude:: ./code_examples/python_create_dataset_azure_example.py
+      :linenos:
+      :language: python
+
 Now we can run the following code to select a subset based on the 
 :code:`'stopping_condition_minimum_distance': 0.1` stopping condition. In a first,
 selection run we only select images with the specific minimum distance between 
```

Comment:
The sentence on line 122 is a bit ambiguous now because it is no longer clear what "the script" is. Maybe rephrase the sentence to make it clear that the dataset creation should not be run again.

Reply:
Actually, I would maybe even copy/paste the script again and increase the `stopping_condition_minimum_distance` to `0.2` (highlight the change!) which should make it very clear what should be run again.

Reply:
nice suggestion! Done